core/src/scss/components/main-nav/_main-nav--dark.scss

```diff
@@ -1,128 +1,139 @@
 @charset "UTF-8";
 
-.su-main-nav {
-  &.su-main-nav--dark {
-    @include grid-media-max('md') {
-      .su-main-nav__toggle {
-        color: $color-white;
-
-        &:hover,
-        &:focus {
-          &::before {
-            background-color: $color-white;
-          }
+.su-main-nav--dark {
+  $_color-map: $main-nav-dark-colors;
```

Comment:
Just in case this variable name is used somewhere else, can you prefix it with the component name. 

eg: `$main-nav-color-map`